tests/drivers/watchdog/wdt_basic_api/src/test_wdt.c

```diff
@@ -78,6 +78,11 @@
 #define TIMEOUTS                   2
 #elif defined(CONFIG_IWDG_STM32)
 #define TIMEOUTS                   0
+#elif defined(CONFIG_WDT_SAM)
+#ifdef CONFIG_SOC_SERIES_SAME70
+#error SAME70 series chip NOT supports this test.
+#endif
```

Comment:
Is this change is required ? as u are spiking the test by adding the same70 to platform exclude in testcase.yaml file

Reply:
@SavinayDharmappa according to #15109. section .noinit be cleared(random) during the boot process on ATSAME70 MCU (SAM E70 Xplained board),  this bug comes from the SoC itself and not from Zephyr.
I have adding the same70 to platform exclude in testcase.yaml file.